test/linters_test.go

```diff
@@ -212,8 +211,6 @@ func extractRunContextFromComments(t *testing.T, sourcePath string) *runContext
 			rc.configPath = configPath
 			continue
 		}
-
-		assert.Fail(t, "invalid prefix of comment line %s", line)
```

Comment:
Why have you removed this line and why do you think it's safe to do so?

Reply:
> Why have you removed this line

I think this line can be removed to allow put comments before `//args`, `//config_path`

> why do you think it's safe to do so?

Why do you think that user comments before  `//args`, `//config_path` is not safe?

Reply:
I would prefer to keep this line and improve condition in line 186. I don't think that comments before two mentioned  prefixes are unsafe but I want this test to fail when it is necessary - and in my opinion you removed rather critical assertion as we expect this code to be never reached (unless tests fail).

Reply:
@iwankgb I do not see any scenario when this assert will really necessary, can you please explain why we need to keep this assert? 

As I know before section `package` for `go` files we can have only comments. So If the source file will have something other before section `package` then it should fail on compile time.

Reply:
If I understand code correctly, then original intention was to `return rc` on multi-line comment that starts with `/*` and hits original line 186. After your change such line will be ignored and would ultimately hit the assertion you deleted. As far as I understand such failure occurs in `test/testdata/go-header.go` and it's perfectly fine with me that multi-line comment would be used for file header although I think that it would be *much* better to add multi-line comment handling to the loot rather then remove `assert.Fail()`.

Reply:
> If I understand code correctly, then original intention was to return rc on multi-line comment that starts with /* and hits original line 

Interesting... Can you point where is using something that starts with multi-line comment? I've checked all testdata/.*\.go files and did not find using this.

Reply:
Here: https://github.com/golangci/golangci-lint/pull/1181/files#diff-787479a653c2e5d026e44d51f6d64580R1

Reply:
> Here: https://github.com/golangci/golangci-lint/pull/1181/files#diff-787479a653c2e5d026e44d51f6d64580R1

Sorry, I thought you meant that multiline comments are used in golangci tests...


>  although I think that it would be much better to add multi-line comment handling to the loop rather then remove assert.Fail().

Yes, it sounds like a solution but I did not quite understand what the case this assert should catch a problem.

Reply:
I'm pretty conservative when it comes to removing testing code and I would appreciate if you handled multi-line comment as all other cases are handled. I do admit that I can't think of non-comment case but I do not know what author had in mind.

Reply:
ok, finally used a more conservative solution.